app/src/main/java/org/wikipedia/dataclient/page/PageSummary.java

```diff
@@ -24,6 +30,8 @@
 
     @Nullable private String type;
     @Nullable private Titles titles;
+    @SuppressWarnings("unused,NullableProblems") @Required
+    @NonNull private String title;
```

Comment:
All fields should be `@Nullable` (and use `defaultString()` in getter methods), because there's no guarantee that deserialized structures will respect nullability.  The `Required` annotation doesn't work exactly like you think it does, and should eventually be removed.

Reply:
Also the `unused` suppression is already applied at the class level.